lighthouse-core/report/html/report-styles.css

```diff
@@ -462,7 +446,7 @@ summary {
 
 .lh-load-opportunity__stats {
   text-align: right;
-  flex: 0 0 var(--lh-audit-score-width);
+  flex: 0 0 calc(5 * var(--body-font-size));
```

Comment:
why the drive by calc now? is the real width not in a var anymore?

Reply:
just a temporary solution till i take on opportunity display. :)